### ISSUE TITLE
Some videos failed to move to /library/calibre-web

### ISSUE THREAD

**Issue by deldesir** [iiab/calibre-web#165]
Likely a timeout issue, which intermittently prevents occasional videos from being properly copied over the filesystem (and likewise their `path` as recorded in xklb-metadata.db).

_Originally posted by @holta in https://github.com/iiab/calibre-web/issues/157#issuecomment-2142570995_

**Comment by holta** [comment]
Potentially both related:

- PR #137 
- #158

**Comment by holta** [comment]
@deldesir is there progress?

Or confirmation this is no longer happening?

🕯️

**Comment by deldesir** [comment]
This should no longer happening as the uploading process has been tightened up with the removal of force-failure on timeout.

**Comment by holta** [comment]
Should we close this ticket later in June 2024 after monitoring many "Download to IIAB" tests ❔

**Comment by deldesir** [comment]
Yes. Meanwhile, I am investigating a case similar to it I am having with the 100th URL of @nzola's IOG academy playlist of playlists. You can see an example live at LRN2 tasks page (the very last download).

**Comment by holta** [comment]
> Yes. Meanwhile, I am investigating a case similar to it I am having with the 100th URL of @nzola's IOG academy playlist of playlists. You can see an example live at LRN2 tasks page (the very last download). This is an xklb issue.

Thanks!!
